web/src/compose.js

```diff
@@ -89,7 +89,7 @@ export function update_video_chat_button_display() {
 export function compute_show_audio_chat_button() {
     const available_providers = page_params.realm_available_video_chat_providers;
     if (
-        (available_providers.jitsi_meet &&
+        (page_params.jitsi_server_url &&
```

Comment:
Would it make sense to leave this as `available_providers.jitsi_meet && page_params.jitsi_server_url`? I know `available_providers.jitsi_meet` is always true, but it seems more logically correct that way.